### ISSUE TITLE
How to Assign few Sections of the Course to Company instead of entire Course

### ISSUE THREAD

**Issue by vishalsat** [iomad/iomad#1626]
Hi,

Is there a way to assign only few sections of the courses to Company instead of assigning the entire course to Company. I am able to assign entire course to Company but did not find the way to assign few sections of the course to company. 

Thank you in advance
Vishal

**Comment by turf212** [comment]
This can be accomplished but you need to use the restrict access core Moodle functionality to achieve it.

You need to assign the course to the company (otherwise there is no potential of their users even accessing the company).
You can then use (< 4.0) restrict access to the sections, activities, resources or what have you using profile field (institution if syncing is enabled under site administration -> plugins -> local -> local IOMAD settings).
If you have 4.0 + you can use the same restrict access but use the company restriction instead.

TLDR
- Set up course and assign to company
- Set up restriction to sections using the restrict access rules
- Enrol users to course.  They can only see the content that is availabel to them
